Source/WebCore/platform/graphics/cairo/ImageBufferCairoBackend.cpp

```diff
@@ -42,12 +42,6 @@
 
 namespace WebCore {
 
-void ImageBufferCairoBackend::clipToMask(GraphicsContext& destContext, const FloatRect& destRect)
-{
-    if (auto image = copyNativeImage(DontCopyBackingStore))
-        Cairo::clipToImageBuffer(*destContext.platformContext(), image->platformImage().get(), destRect);
```

Comment:
Humorously, Cairo had already overridden this on GraphicsContextCairo, but still had this implementation here too. (That's why you don't see me adding anything to GraphicsContextCairo, though).